core/src/test/java/uk/gov/gchq/koryphe/impl/function/CallMethodTest.java

```diff
@@ -99,13 +99,13 @@ public void shouldJsonSerialiseAndDeserialise() throws IOException {
     }
 
     @Override
-    protected Function getInstance() {
+    protected CallMethod getInstance() {
         return new CallMethod();
     }
 
     @Override
-    protected Class<? extends Function> getFunctionClass() {
-        return CallMethod.class;
+    protected Iterable<CallMethod> getDifferentInstances() {
```

Comment:
What would you suggest changing it to? The ones that return null are ones which have no variety ie: ToLong. Ones that return single instances tend to be ones with only a single parameter. Hence why I used an iterable to capture as many possibilities.

Reply:
Perhaps it's just the method name that needs to change. I suppose as long as you're happy there's no chance for gotchas in future it can remain. It's situations where people calling the method expect multiple instances I was worried about.

Reply:
I'll add some Javadoc to the method to explain it